lib/parser.rb

```diff
@@ -46,6 +46,7 @@ module Source
     require 'parser/source/map/variable'
     require 'parser/source/map/keyword'
     require 'parser/source/map/definition'
+    require 'parser/source/map/endless_definition'
```

Comment:
I actually think that someday MRI developers may introduce endless classes/methods, so maybe we'll reuse it:
```rb
class A = def m = 42
module M = def m = 42
```